sdk/azcore/version.go

```diff
@@ -10,5 +10,5 @@ const (
 	UserAgent = "azcore/" + Version
 
 	// Version is the semantic version (see http://semver.org) of this module.
-	Version = "v0.16.3"
+	Version = "v0.17.0"
```

Comment:
v0.17.0 is the version I already released. 
```suggestion
	Version = "v0.18.0"
```